wdtk-wikibaseapi/src/main/java/org/wikidata/wdtk/wikibaseapi/WbEditingAction.java

```diff
@@ -123,9 +125,10 @@
 	 *            prefix of entity URIs), e.g.,
 	 *            "http://www.wikidata.org/entity/"
 	 */
-	public WbEditEntityAction(ApiConnection connection, String siteIri) {
+
+	public WbEditingAction(ApiConnection connection, String siteUri) {
 		this.connection = connection;
-		this.siteIri = siteIri;
+		this.siteIri = siteUri;
```

Comment:
Why this renaming?

Reply:
That's inherited from the Jackson implementation - I will fix this.